libmamba/include/mamba/fs/filesystem.hpp

```diff
@@ -73,11 +73,16 @@ namespace mamba::fs
     {
     };
 
+    struct Utf8Options
+    {
+        bool normalize_sep = true;
+    };
+
     // Maintain `\` on Windows, `/` on other platforms
     std::filesystem::path normalized_separators(std::filesystem::path path);
 
     // Returns a UTF-8 string given a standard path.
-    std::string to_utf8(const std::filesystem::path& path);
+    std::string to_utf8(const std::filesystem::path& path, Utf8Options utf8_options);
```

Comment:
Consider defaulting to simplify usage
```suggestion
    std::string to_utf8(const std::filesystem::path& path, Utf8Options utf8_options = {});
```